examples/gno.land/r/demo/grc20factory/grc20factory.gno

```diff
@@ -137,17 +128,40 @@ func TransferInstanceOwnership(cur realm, symbol string, newOwner std.Address) {
 	checkErr(inst.admin.TransferOwnership(newOwner))
 }
 
+func ListTokens() []*grc20.Token {
```

Comment:
```suggestion
func ListTokens(page, limit int) []*grc20.Token {
```

Reply:
Ok I'll add pagination on this 👍

Reply:
Done: https://github.com/gnolang/gno/pull/4373/commits/a3077532a64949a8f1f4d3774fb589ff12fb7696